src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$header-height: 60px;
+$logo-heigth: 40px;
+$card-width: 200px;
+$card-height: 300px;
+
+$color-border: #ccc;
+$color-nav__item: #060b35;
```

Comment:
names of your variables should be more descriptive
If we speak of colors - those should be names something like `light-blue` or `light-grey` or something like that

Reply:
changed them 👍